src/posts/posts.module.ts

```diff
@@ -1,7 +1,11 @@
 import { Module } from '@nestjs/common';
+import { PostsController } from './posts.controller';
+import { PostService } from './posts.service';
+import { PrismaModule } from 'nestjs-prisma';
 
 @Module({
-  imports: [],
-  providers: [],
+  imports: [PrismaModule],
```

Comment:
We are declaring PrismaModule as a global module on the `AppModule.ts`. So I think it's OK to remove this import.